package.json

```diff
@@ -38,7 +38,7 @@
 		"*.{js,ts,svelte,css,md,json}": [
 			"prettier --write",
 			"eslint --fix",
-			"tsc --noEmit"
+			"bash -c 'yarn run check'"
```

Comment:
the why: https://github.com/lint-staged/lint-staged/issues/825#issuecomment-674575655

Reply:
why do you need bash -c, would"yarn run check" fail on its own?